dev/conductor/README.md

```diff
@@ -4,14 +4,14 @@ Command-line tool for managing a release of the Flutter SDK. Also see
 [Release Process](../../docs/releases/Release-process.md) for more information on
 the release process.
 
-## Requirements
+## Conductor Requirements
 
-Some basic requirements to conduct a release are:
+Some basic requirements to run conductor are:
 
 - a Linux or macOS computer set up for Flutter development. The conductor does
   not support Windows.
 - git
-- Mirrors on GitHub of the Flutter
+- Mirrors on GitHub of the Flutter **
```

Comment:
This seems like an incomplete phrase. Is this supposed to be "of the following Repos"?

Reply:
oh wait, nvm